src/promptflow/promptflow/executor/_process_manager.py

```diff
@@ -225,6 +226,7 @@ def start_processes(self):
             ),
         )
         process.start()
+        return process.pid
```

Comment:
Manager process id should not be returned here, because it's not consistent with the interface of SpawnProcessManager anymore.
Consider set `self.spawned_process_manager_pid`, then the caller can get it by fork_process_manager.spawned_process_manager_pid